mrp_operations_traceability/README.rst

```diff
@@ -0,0 +1,21 @@
+Traceability for production operations
+======================================
+
+This module allows you to manage the complete traceability of products consumed
+in production operations.
+
+To view this complete traceability of a lot, this module shows the "Full
```

Comment:
Gracias por la explicación. Dos cosas entonces:
* Cambia el texto y traducción del botón para el otro módulo (en este mismo PR).
* Cambia este descripción, que indica que este módulo añade el botón, y pon mejor: "You can access the full traceability of a lot from the button "Full traceability" in the lot form."